### PR TITLE
[staging-next] changedetection-io: 0.39.22.1 -> 0.40.3

### PR DESCRIPTION
**Pull request by K900**
###### Description of changes

Ew.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/flask-expects-json/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "flask-expects-json";
+  version = "1.7.0";
```
Comment (mweinelt) [inline pkgs/development/python-modules/flask-expects-json/default.nix:10]:
```suggestion
  version = "1.7.0";
  format = "setuptools";
```

We try to make format a required attribute.

File: pkgs/development/python-modules/flask-expects-json/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-TvGGqG8QVyohr4LlSVRt7aAkMmYotKls0U06D3F1TWI=";
+  };
+
+  propagatedBuildInputs = [ flask jsonschema ];
+
+  # tests aren't included in sdist and don't pass on current flask
```
Comment (mweinelt) [inline pkgs/development/python-modules/flask-expects-json/default.nix:19]: Then fetch git instead, because we will want to run those eventually.

Comment (K900) [reply pkgs/development/python-modules/flask-expects-json/default.nix:22]: Done.

File: pkgs/development/python-modules/flask-expects-json/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    owner = "Fischerfredl";
+    repo = pname;
+    rev = version;
+    hash = "sha256-CUxuwqjjAb9Fy6xWtX1WtSANYaYr5//vY8k89KghYoQ=";
+  };
+
+  propagatedBuildInputs = [ flask jsonschema ];
+
+  # tests aren't included in sdist and don't pass on current flask
+  doCheck = false;
```
Comment (fabaff) [inline pkgs/development/python-modules/flask-expects-json/default.nix:23]: It's possible to execute some tests, see https://github.com/NixOS/nixpkgs/pull/218018/commits/65f3d3bd9c8f196afb0b4baeccbd499ebaba187d

Comment (mweinelt) [reply pkgs/development/python-modules/flask-expects-json/default.nix:23]: @SuperSandro2000 What abou this review comment?

Comment (K900) [reply pkgs/development/python-modules/flask-expects-json/default.nix:23]: Pushed as a follow-up change in #220613.

**Pull request merged**